notebooks/energy_volume_curve.ipynb

```diff
@@ -30,14 +30,28 @@
     "ExecuteTime": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pr = Project(path='thermo')
</pre><p>Nit: I would omit the keyword, especially because you explicitly call it the "name" above (which I think is a fair term, we don't need to be 100% consistent on that, it's just mildly jarring to see both terms so close together)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron/pull/1479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>